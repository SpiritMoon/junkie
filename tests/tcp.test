#!/bin/sh

NAME="TCP"
PCAPS="pcap/postgres/connect.pcap"
filter() {
	# We are only interested in TCP header decoding (and especialy options)
	grep '^TCP' | sed -e 's/^\([^@]\+\)@[0-9a-fx]\+\(.*\)$/\1\2/'
}
OUTPUT="tcp.out"
CMDLINE='(set-log-level log-debug)'

. "$srcdir/dump_test"
