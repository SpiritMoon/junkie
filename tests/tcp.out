-- pcap/postgres/connect.pcap --
TCP: head_len=40, payload=0, ports=50076->5432(srv), flags=Syn, win=5840, ack=0, seq=3085811749, urg=0, opts=MSS(1460),4,8,nop,WSF(7)
TCP: head_len=40, payload=0, ports=5432(srv)->50076, flags=SynAck, win=65160, ack=3085811750, seq=429171150, urg=0, opts=MSS(1460),4,8,nop,WSF(11)
TCP: head_len=32, payload=0, ports=50076->5432(srv), flags=Ack, win=46, ack=429171151, seq=3085811750, urg=0, opts=nop,nop,8
TCP: head_len=32, payload=34, ports=50076->5432(srv), flags=AckPsh, win=46, ack=429171151, seq=3085811750, urg=0, opts=nop,nop,8
TCP: head_len=32, payload=0, ports=5432(srv)->50076, flags=Ack, win=32, ack=3085811784, seq=429171151, urg=0, opts=nop,nop,8
TCP: head_len=32, payload=13, ports=5432(srv)->50076, flags=AckPsh, win=32, ack=3085811784, seq=429171151, urg=0, opts=nop,nop,8
TCP: head_len=32, payload=0, ports=50076->5432(srv), flags=Ack, win=46, ack=429171164, seq=3085811784, urg=0, opts=nop,nop,8
TCP: head_len=32, payload=41, ports=50076->5432(srv), flags=AckPsh, win=46, ack=429171164, seq=3085811784, urg=0, opts=nop,nop,8
TCP: head_len=32, payload=303, ports=5432(srv)->50076, flags=AckPsh, win=32, ack=3085811825, seq=429171164, urg=0, opts=nop,nop,8
TCP: head_len=32, payload=0, ports=50076->5432(srv), flags=Ack, win=54, ack=429171467, seq=3085811825, urg=0, opts=nop,nop,8
TCP: head_len=32, payload=5, ports=50076->5432(srv), flags=AckPsh, win=54, ack=429171467, seq=3085811825, urg=0, opts=nop,nop,8
TCP: head_len=32, payload=0, ports=50076->5432(srv), flags=AckFin, win=54, ack=429171467, seq=3085811830, urg=0, opts=nop,nop,8
TCP: head_len=32, payload=0, ports=5432(srv)->50076, flags=AckFin, win=32, ack=3085811831, seq=429171467, urg=0, opts=nop,nop,8
TCP: head_len=32, payload=0, ports=50076->5432(srv), flags=Ack, win=54, ack=429171468, seq=3085811831, urg=0, opts=nop,nop,8
